### PR TITLE
Internal API to filter commands based on input

### PR DESCRIPTION
**Pull request by DonJayamanne**
For #13892

### CODE REVIEW

File: src/notebooks/controllers/kernelSource/remoteNotebookKernelSourceSelector.ts [hunk 1]
```diff
@@ -213,7 +226,32 @@ export class RemoteNotebookKernelSourceSelector implements IRemoteNotebookKernel
...
+                    const quickPickCommandItems = [];
+                    if (quickPickServerItems.length > 0) {
+                        quickPickCommandItems.push({
+                            label: Common.labelForQuickPickSeparatorIndicatingThereIsAnotherGroupOfMoreItems,
+                            kind: QuickPickItemKind.Separator
+                        });
+                    }
 
+                    const commands = await provider.getQuickPickEntryItems(e);
+                    const newProviderItems: KernelProviderItemsQuickPickItem[] = commands.map((i) => {
```
Comment (amunger) [inline src/notebooks/controllers/kernelSource/remoteNotebookKernelSourceSelector.ts:243]: is there something special about this group that can be described better than 'more'?

**Review by amunger — APPROVED** [review]

File: src/notebooks/controllers/kernelSource/remoteNotebookKernelSourceSelector.ts (hunk 1 above)
Comment (amunger) [reply src/notebooks/controllers/kernelSource/remoteNotebookKernelSourceSelector.ts:243]: (I'm not really sure what it currently looks like)

Comment (DonJayamanne) [reply src/notebooks/controllers/kernelSource/remoteNotebookKernelSourceSelector.ts:243]: This is existing code. More iss Jessy a separator, blue line

Comment (DonJayamanne) [reply src/notebooks/controllers/kernelSource/remoteNotebookKernelSourceSelector.ts:243]: Will demo next week

**Pull request merged**